README.md

```diff
@@ -1 +1,114 @@
-# Leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderboard] <a name="about-project"></a>
+
+**[Leaderboard]** is a project that displays scores submitted by different players. It also allows a user to submit their own score. All data is preserved by the external leaderboard API service.
+
+## 🛠 Built With <a name="built-with">
+Webpack
+</a>
+
+### Tech Stack <a name="tech-stack">
+Leaderboard API
+JavaScript
+CSS3
+HTML
+</a>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+ - IDE(Code editor) eg VScode
+ - Git
+### Setup
+
+Clone this repository to your desired folder:
+ - git clone https://github.com/willy836/Leaderboard
+ - cd Leaderboard/
+ - code .
+ - open dist/index.html with live server
+
+### Install
+
+Install this project with:
+ - linters and other necessary dependencies (npm install)
+
+### Usage
+This project can be used by anyone who wants to learn Webpack, JavaScript, and how to use an external API service
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Wilson Kimiri**
+
+- GitHub: [@githubhandle](https://github.com/willy836)
+- Twitter: [@twitterhandle](https://twitter.com/Kimiri836)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/wilson-kimiri-420396235/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] **[OPTIONAL]:** You did a great job documenting your project! 🚀 However, it would be great if you include the actual link to your repo's issues so people that want to collaborate on your project can have an easy way to create a new issue. 🤓 🛠️